js/background.js

```diff
@@ -1491,16 +1498,17 @@ function getDebugInfo(callback) {
         return the_debug_info;
     }
 
-    if (!SAFARI) {
+    if (!SAFARI && !EDGE) {
```

Comment:
Could simplify to !(SAFARI || EDGE) by De Morgan's Law. Might save a single CPU cycle? 😛